RockPaperScissors/RockPaperScissors/Game.swift

```diff
@@ -27,10 +27,10 @@ struct Game {
         }
     }
```

Comment:
이전 스텝이 머지되기 전에 커밋된 부분들이 있어서 코멘트가 안찍히네용ㅠㅠ 임시로 이부분에 코멘트 찍습니당!
```swift
struct Game {
```
struct를 사용해보셨네요! 너무 좋습니다👍👍
class대신 struct를 사용한 이유는 무엇일까요?
class와 struct의 간단한 차이점도 설명해주세요!

Reply:
구조체는 값 형식이고, 인스턴스 데이터를 모두 스택에 저장합니다.
클래스는 참조형식이고, 인스턴스 데이터를 힙에 저장하고, 해당 힙을 가리키는 변수를 스택에 저장합니다.

저희가 짠 구조 상으로는 상속이 필요하지 않습니다.
상속이 필요하지 않다면 구조체를 쓰는 게 메모리 효율 측면에서 더 좋기 때문에 구조체를 사용했습니다.

이와 관련, 아래의 공식문서의 문장을 인용합니다.
[출처: Structures and Classes -> Comparing Structures and Classes](https://docs.swift.org/swift-book/LanguageGuide/ClassesAndStructures.html)
> The additional capabilities that classes support come at the cost of increased complexity. As a general guideline, prefer structures because they’re easier to reason about, and use classes when they’re appropriate or necessary. In practice, this means most of the custom data types you define will be structures and enumerations. For a more detailed comparison, see [Choosing Between Structures and Classes](https://developer.apple.com/documentation/swift/choosing_between_structures_and_classes).